manifold/src/impl.cpp

```diff
@@ -449,18 +449,20 @@ void Manifold::Impl::CreateHalfedges(const VecDH<glm::ivec3>& triVerts) {
   // drop the old value first to avoid copy
   halfedge_.resize(0);
   halfedge_.resize(3 * numTri);
-  VecDH<TmpEdge> edge(3 * numTri);
+  VecDH<uint64_t> edge(3 * numTri);
+  VecDH<int> ids(3*numTri);
```

Comment:
It looks like our code formatting check is missing the spacing here. Any idea why?

Reply:
https://google.github.io/styleguide/cppguide.html#Horizontal_Whitespace

Operator:

```
// Other binary operators usually have spaces around them, but it's
// OK to remove spaces around factors.  Parentheses should have no
// internal padding.
```

maybe we should set it explicitly that these operators should have spaces around them.

Reply:
I found that there is no options in clang-format to force that.

Reply:
That's strange, since clang-format automatically changes that kind of spacing. I sort of thought the check just ran auto-format on the files and checked that it was a no-op.

Reply:
The action is just a check, because I don't think it is a good idea to make the bot commit automatically. That will mess up git blame.

So I guess clang-format will probably change that spacing, but it will not treat that as an error.